doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -10,9 +10,36 @@
 
 <a name="6.1"></a>
 ## 6.1 Introduction
+The adoption of cloud-native allows development teams to move fast, deploy software efficiently, and operate at an unprecedented scale. 
+Cloud-native applications offer the benefits by providing applications as small microservices (either containers or serverless), and managing them through DevOps processes. To build these applications, organisations need a cloud-native infrastructure which provides flexibility and enough services for developers. Cloud-native infrastructures should provide a platform for the build and release team to effectively perform continuous delivery of its applications, with close to zero human intervention. With Cloud-native, the industy requires a radically different apporach to security in contrast to monolithic architecture. 
```

Comment:
Cloud-native -> Cloud native (no hyphen, as per CNCF style guide: https://github.com/cncf/foundation/blob/master/style-guide.md).  Other instances later in the paragraph.

Reply:
Fixed.